src/test.js

```diff
@@ -537,18 +544,31 @@ function checkPollution() {
 
 // Will be exposed as QUnit.asyncTest
 function asyncTest( testName, expected, callback ) {
+	var warning;
+
+	if ( global.console && global.console.warn ) {
+		warning = "DEPRECATED: QUnit.asyncTest() will be removed in QUnit 2.0.\n" +
+			"Reference: https://api.qunitjs.com/QUnit.asyncTest/";
+		global.console.warn(warning);
```

Comment:
I try implement this idea in aee09a2b58bd9f84e2c95d0c92ba6c37ebdcb65f, what do you think?